data/docs.yml

```diff
@@ -90,20 +90,23 @@
 
     - name: Python Agent
       icon: python-agent
-      description: The Python Agent for Apache SkyWalking, which provides the native tracing/logging abilities for Python projects.
+      description: The Python Agent for Apache SkyWalking, which provides the native tracing/metrics/logging/profiling abilities for Python projects.
       user: apache
       repo: skywalking-python
       repoUrl: https://github.com/apache/skywalking-python.git
       docs:
         - version: Next
           link: /docs/skywalking-python/next/readme/
+        - version: v1.0.0
+          link: /docs/skywalking-python/next/readme/
+          commitId: 6f745cdbec7cf003c60d71857bf987a280e88430
         - version: v0.8.0
```

Comment:
As 1.0 is a GA, do we still plan to keep 0.8? Usually previous versions are kept aftrr GA.

Reply:
You replaced download page, but keep docs

Reply:
Oh I was confused why Java agent has previous versions while others don't. I will add it back.

Reply:
It is stable, and many ppl are keeping in some old releases. I think Python would face that day as you have tagged GA.